index.html

```diff
@@ -12,40 +12,289 @@
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">Ruby</button></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button type="button" class="">See Project </button>
+                    </div>
+                </div>
+                <div class="card-container">
+                    <div class="work-card">
+                        <div class="work-card-body">
+                            <h1>Professional Art print Data</h1>
+                            <p class="wprk-description">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                                industry's standard
+                            </p>
+                            <div class="work-tech-stack">
+                                <ul class="ul-links">
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">html</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">bootstrap</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">Ruby</button></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button  type="button" class="">See Project </button>
+                    </div>
+                </div>
+                <div class="card-container">
+                    <div class="work-card">
+                        <div class="work-card-body">
+                            <h1>Professional Art print Data</h1>
+                            <p class="wprk-description">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                                industry's standard
+                            </p>
+                            <div class="work-tech-stack">
+                                <ul class="ul-links">
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">html</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">bootstrap</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">Ruby</button></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button type="button" class="">See Project </button>
```

Comment:
- [ ]  Kindly add hover effect to button so it looks like button in the figma design